libs/e2e/src/util/comboBox.ts

```diff
@@ -20,3 +20,19 @@ export function selectComboBox(selector: string, item: string | number): void {
 export function getComboBoxValue(selector: string): string | undefined {
   return cy.$$(`${selector} input.ms-ComboBox-Input`).val()?.toString();
 }
+
+export function multiSelectComboBox(selector: string, item: string | number, firstSelect?: boolean): void {
```

Comment:
Thanks for adding this!

Reply:
Sure! Strangely, the `click` is only necessary on the first call and not on further ones. Couldn't quite figure out why, but I guess it doesn't matter as long as it works as intended.